src/graph/optimizer/rule/CollapseProjectRule.h

```diff
@@ -13,16 +13,36 @@ DECLARE_bool(enable_optimizer_collapse_project_rule);
 namespace nebula {
 namespace opt {
 
-/**
- * Combines two [[Project]] nodes into one
- * Required conditions:
- *  1. Match the pattern
- *  2. Expressions between nodes cannot be referenced more than once
- * Benefits:
- *  1. reduce the copy of memory between nodes
- *  2. reduces expression overhead in some cases(similar to column pruning)
- */
+/*
```

Comment:
I believe we should use `//`

Reply:
Any reference?
Google style?

Reply:
Google style doesn't declare that the comments should use `//`, but it suggests that the comments style should be unified. I think it makes sense. And the most open source projects i had ever seen use the `//`. For now i have seen much styles in our project, it's too ugly.
```
// 

/*

*/

/**
 *
 */
```

Reply:
ACK.